readthedocs/rtd_tests/tests/test_footer.py

```diff
@@ -448,7 +449,7 @@ def test_highest_version_without_tags(self):
 
     def test_private_highest_version(self):
         self.pip.versions.update(privacy_level=PRIVATE)
-        owner = self.pip.users.first()
+        owner = AdminPermission.members(self.pip).first()
```

Comment:
Hrm... This returns `[]`

We need to create an organization, make the user owner/member and add the project to that organization to make this test work on commercial.

@stsewd would you mind updating the test here?

Reply:
With that change, it works fine:

```
(Pdb) o = get(Organization)
(Pdb) o
<Organization: 1>
(Pdb) o.projects.add(project)
(Pdb) o.owners.add(user)
(Pdb) get_version_compare_data(self.pip, base_version, user=owner)
{'project': 'Version 0.8.1 of Pip (19)', 'version': '0.8.1', 'is_highest': False, 'url': 'https://pip.readthedocs.io/en/0.8.1/', 'slug': '0.8.1'}
(Pdb)
```